src/uu/chroot/src/chroot.rs

```diff
@@ -94,22 +106,20 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         );
     }
 
-    let command: Vec<&str> = match matches.args.len() {
+    let commands = match matches.values_of(options::COMMAND) {
+        Some(v) => v.map(|v| v).collect(),
+        None => vec![],
+    };
+
+    let command: Vec<&str> = match commands.len() {
```

Comment:
Since there is only one command, would it make sense to get the commands and args independently via clap? That might be a little be cleaner, but would also require a bit more of refactoring, so I guess this is fine for now.

Reply:
I agree that matching `chroot [OPTION] NEWROOT [COMMAND [ARG]...]` with  `option::COMMAND` and `option::ARGS` would be cleaner code. If @ycd wants to take it on... ? Otherwise, I'd just request an additional FixME/ToDO comment to prompt someone else to take at a later date.

Thanks for the work, @ydc.

Reply:
Do we need another `ARGS`? 

The syntax for GNU coreutils looks like this: 

```
chroot [-u -user] [-g -group] [-G -group,group,...] newroot [command]
```

Isn't the ARGS and COMMANDS are the same thing?

Reply:
The idea was to match them separately so that you don't have to treat the first element of the `commands` vec differently. Like what happens here:
```rust
let pstatus = Command::new(command[0])
        .args(&command[1..])
        .status()
.unwrap_or_else(|e| crash!(1, "Cannot exec: {}", e));
```
But like rivy said, this is great and there's no need to do this refactor now (or maybe it doesn't even lead to cleaner code because the `default_shell` thing is tricky).

Reply:
Just FYI, my Ubuntu `chroot` usage message is:

```text
Usage: chroot [OPTION] NEWROOT [COMMAND [ARG]...]
  or:  chroot OPTION
Run COMMAND with root directory set to NEWROOT.
...
```

Reply:
Ah, i see! Thanks for explanations, I'm just gonna add a ToDo comment then.